app/src/main/java/com/keylesspalace/tusky/fragment/SFragment.java

```diff
@@ -75,9 +71,6 @@
 
 import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers;
 import kotlin.Unit;
-import retrofit2.Call;
-import retrofit2.Callback;
-import retrofit2.Response;
```

Comment:
There are even more unused imports, and `TAG` is also no longer used.

But moving out things from SFragment, yeah! This thing has to die.